### PR TITLE
Create first state for development

### PR DESCRIPTION
**Pull request by UDOMPOL-T**

### CODE REVIEW

**Comment by SuebyartP** [comment]
Title not makesense

**Comment by UDOMPOL-T** [comment]
> Title not makesense

Nah bro It make sense

File: mineinfo/src/pages/index.astro [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (SuebyartP) [inline mineinfo/src/pages/index.astro:11]: Index or Document ?

File: mineinfo/src/pages/index.astro [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+---
+
+
+
```
Comment (SuebyartP) [inline mineinfo/src/pages/index.astro:5]: Too many space

**Pull request merged**